### PR TITLE
2005

### PR DESCRIPTION
**Pull request by pdeiml**
Ein paar Kleinigkeiten in den jeweiligen Commits.

### CODE REVIEW

File: input/data/2005/2005A%26A...439.1013A/tev-000061.yaml [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+  sigma: {val: 0.16, err: 0.02}
+  sigma2: {val: 0.16, err: 0.02}
+  #TODO: no pa given in the paper
+
+spec:
+  type: pl
+
+  norm: {val: 4.3, err: 0.3, err_sys: }
+  index: {val: 2.4, err: 0.05, err_sys: 0.2}
+  ref: 1
```
Comment (cdeil) [inline input/data/2005/2005A%26A...439.1013A/tev-000061.yaml]: Add newline at end of file.

File: input/data/2005/2005A%26A...439.1013A/tev-000061.yaml [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+
+pos:
+  ra: {val: 13h3m0.4s, err: 4.4s, err_sys: 20s}
+  dec: {val: -63d11m55s, err: 31s, err_sys: 20s}
+
+morph:
+  type: gauss
+
+  sigma: {val: 0.16, err: 0.02}
+  sigma2: {val: 0.16, err: 0.02}
```
Comment (cdeil) [inline input/data/2005/2005A%26A...439.1013A/tev-000061.yaml]: Remove `sigma2` and `TODO` comment for `pa`.
(this was a symmetric gaussian fit and there's just one extension `sigma`.

File: input/data/2005/2005A%26A...439.1013A/tev-000061.yaml [hunk 3]
```diff
@@ -1,4 +1,25 @@
...
+  significance: 21
+
+pos:
+  ra: {val: 13h3m0.4s, err: 4.4s, err_sys: 20s}
+  dec: {val: -63d11m55s, err: 31s, err_sys: 20s}
+
+morph:
+  type: gauss
+
+  sigma: {val: 0.16, err: 0.02}
```
Comment (cdeil) [inline input/data/2005/2005A%26A...439.1013A/tev-000061.yaml]: Suggest to put `0.16d` here to be explicit about degrees.
I think you had a file earlier where you gave a `sigma` without units, and it was in `arcmin`.
I don't even know what our default is, best to be explicit in the input data files.

File: input/data/2005/2005A%26A...439.1013A/tev-000061.yaml [hunk 4]
```diff
@@ -1,4 +1,25 @@
...
 reference_id: 2005A&A...439.1013A
 
-# TODO: add data
+data:
+  livetime: 58.6
+  excess: 2469
+  significance: 21
+
+pos:
+  ra: {val: 13h3m0.4s, err: 4.4s, err_sys: 20s}
```
Comment (cdeil) [inline input/data/2005/2005A%26A...439.1013A/tev-000061.yaml]: `err: 0h0m4.4s`

File: input/data/2005/2005A%26A...439.1013A/tev-000061.yaml [hunk 5]
```diff
@@ -1,4 +1,25 @@
...
+
+  sigma: {val: 0.16, err: 0.02}
+  sigma2: {val: 0.16, err: 0.02}
+  #TODO: no pa given in the paper
+
+spec:
+  type: pl
+
+  norm: {val: 4.3, err: 0.3, err_sys: }
+  index: {val: 2.4, err: 0.05, err_sys: 0.2}
```
Comment (cdeil) [inline input/data/2005/2005A%26A...439.1013A/tev-000061.yaml]: `val: 2.44` (given in sect 4.4)

File: input/data/2005/2005A%26A...442....1A/tev-000060.yaml [hunk 6]
```diff
@@ -1,4 +1,21 @@
 source_id: 60
 reference_id: 2005A&A...442....1A
 
-# TODO: add data
+data:
+  livetime: 48.6
+  significance: 13.8
+
+pos:
+  ra: {val: 13h2m49.3s, err: 2.3s, err_sys: }
```
Comment (cdeil) [inline input/data/2005/2005A%26A...442....1A/tev-000060.yaml]: `err: 0h0m3.6s`

File: input/data/2005/2005A%26A...442....1A/tev-000060.yaml [hunk 7]
```diff
@@ -1,4 +1,21 @@
...
+  dec: {val: -63d49m53s, err: 17s, err_sys: }
```
Comment (cdeil) [inline input/data/2005/2005A%26A...442....1A/tev-000060.yaml]: `val: -63d50m2.4s, err: 21.1s`

File: input/data/2005/2005A%26A...442....1A/tev-000060.yaml [hunk 8]
```diff
@@ -1,4 +1,21 @@
...
+
+morph:
+  type: gauss
```
Comment (cdeil) [inline input/data/2005/2005A%26A...442....1A/tev-000060.yaml]: This source (`source_id: 60`) is PSRB1259 and morph type should be "point".
> The width of the PSR B1259-63/SS 2883 ex- cess was found to be compatible with the width of a point- source

File: input/data/2005/2005A%26A...442....1A/tev-000060.yaml [hunk 9]
```diff
@@ -1,4 +1,21 @@
...
+
+spec:
```
Comment (cdeil) [inline input/data/2005/2005A%26A...442....1A/tev-000060.yaml]: I didn't find the `spec` measurement for PSR B1259 in the paper.
So I'm not sure where you got those number from.
Remove?

Comment (pdeiml) [inline input/data/2005/2005A%26A...442....1A/tev-000060.yaml]: In the abstract and on page 4, right side in the middle.
Does pl need a erange or is it optional?

Comment (cdeil) [inline input/data/2005/2005A%26A...442....1A/tev-000060.yaml]: Energy range is optional. Add it if it's given, and if not, just leave it out.

Comment (pdeiml) [inline input/data/2005/2005A%26A...442....1A/tev-000060.yaml]: Do you agree with the spectrum?

**Comment by pdeiml** [comment]
Rechecked the errors.

Sofern du mit dem Spectrum in 

https://github.com/gammapy/gamma-cat/pull/81/files/a25c2aabfb0833eeb961d432a632dc12f4299c5b#diff-5c0b61593f32bd9c777727d3c1068d46

einverstanden bist (see inline comment):

Ready to merge!

**Comment by cdeil** [comment]
Like I said, I didn't find the spectrum info in the paper.
But probably OK, I don't have time to look again.
Merging this now.
Thanks!

**Pull request merged**